EIPS/eip-draft_AUTH_USURP.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: Replace Externally Owned Accounts with `AUTHUSURP`
+description: Allow migrating away from ECDSA by deploying code in place of an externally owned account.
+author: Dan Finlay (@danfinlay), Sam Wilson (@SamWilsn)
+discussions-to: <URL>
```

Comment:
Will need a `discussions-to` before merging.

Reply:
```suggestion
discussions-to: [ethereum-magicians](https://ethereum-magicians.org/t/eip-5003-auth-usurp-publishing-code-at-an-eoa-address/8979)
```